ci/builders/linux_host_engine.json

```diff
@@ -224,6 +224,11 @@
                         "--engine-capture-core-dump"
                     ],
                     "script": "flutter/testing/run_tests.py"
+                },
+                {
+                    "language": "bash",
+                    "name": "Generate metrics test",
+                    "script": "flutter/testing/benchmark/generate_metrics.sh"
```

Comment:
Changes to the upload script are not otherwise tested except by the dry-run here: https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine/engine_metrics.py#69, which we should retain.

Reply:
Added the upload as dry-run.